include/tango/client/device/device_command_info.hxx

```diff
@@ -0,0 +1,82 @@
+//
```

Comment:
When you create a new file, try to keep the standard Tango comments at the top of the file: 
```
//
// This file is part of Tango.
// etc.
```